workbench-core/accounts/src/services/costCenterService.test.ts

```diff
@@ -115,6 +118,99 @@ describe('CostCenterService', () => {
     });
   });
 
+  describe('list costCenters', () => {
+    const costCenterId = 'cc-someId';
+    const costCenterJson = {
+      pk: `CC#${costCenterId}`,
+      sk: `CC#${costCenterId}`,
+      id: costCenterId,
+      name: 'CostCenter-2',
+      // dependency: 'acc-2272f4a8-1791-419a-8e49-b926f3eb46f9',
+      dependency: accountId,
+      description: 'Description for CostCenter-2',
+      subnetId: accountMetadata.subnetId,
+      vpcId: accountMetadata.vpcId,
+      envMgmtRoleArn: accountMetadata.envMgmtRoleArn,
+      externalId: accountMetadata.externalId,
+      encryptionKeyArn: accountMetadata.encryptionKeyArn,
+      environmentInstanceFiles: accountMetadata.environmentInstanceFiles,
+      hostingAccountHandlerRoleArn: accountMetadata.hostingAccountHandlerRoleArn,
+      awsAccountId: accountMetadata.awsAccountId,
+      createdAt: mockDateObject.toISOString(),
+      updatedAt: mockDateObject.toISOString()
+    };
+    let expectedCostCenter: { [key: string]: unknown } = {
+      ...costCenterJson,
+      accountId: costCenterJson.dependency
+    };
+    expectedCostCenter = removeDynamoDbKeys(expectedCostCenter);
+
+    describe('with more than one "page" of costCenters', () => {
+      test('it return costCenters with a pagination token', async () => {
+        // BUILD
+        // Build request for system under test
+        const request = ListCostCentersRequestParser.parse({
+          pageSize: '1',
+          filter: { name: { begins: 'CostCenter' } },
+          sort: { name: 'desc' }
+        });
+
+        // Mock out DDB Service call. DDB service return JSON data and paginationToken
+        const paginationToken = 'exampleToken';
+        jest.spyOn(DynamoDBService.prototype, 'getPaginatedItems').mockImplementation((param) => {
+          expect(param).toEqual({
+            key: { name: 'resourceType', value: 'costCenter' },
+            index: 'getResourceByName',
+            limit: 1,
+            sortKey: 'name',
+            begins: { S: 'CostCenter' },
+            forward: false
+          });
+          return Promise.resolve({
+            data: [costCenterJson],
+            paginationToken
+          });
+        });
+
+        // OPERATE & CHECK
+        await expect(costCenterService.listCostCenters(request)).resolves.toEqual({
+          data: [expectedCostCenter],
+          paginationToken: 'exampleToken'
+        });
+      });
+    });
+
+    describe('with one cost center', () => {
+      test('it returns one cost center', async () => {
```

Comment:
Do we want to be testing that there is one cost center, or that no pagination token is returned when there is less than `pageSize` results?